s3_sync/sync_s3.py

```diff
@@ -853,6 +853,12 @@ def _get_error_message(self, error, op, args):
             parts.append(error.message)
         return ' '.join(parts)
 
+    def select_container_metadata(self, metadata_dict):
+        return {}
+
+    def post_container(self, metadata):
+        pass
```

Comment:
I'm guessing this is the provider that returns None. Can we return a canned failure here, like:
```suggestion
         return ProviderResponse(False, 501, {}, '')
```